pkg/credentialprovider/plugin/plugin.go

```diff
@@ -98,7 +98,10 @@ func RegisterCredentialProviderPlugins(pluginConfigFile, pluginBinDir string) er
 	registerMetrics()
 
 	for _, provider := range credentialProviderConfig.Providers {
-		pluginBin := filepath.Join(pluginBinDir, provider.Name)
+		pluginBin, err := exec.LookPath(filepath.Join(pluginBinDir, provider.Name))
```

Comment:
could you also add comments here, and also it's better having a ut coverage for this. thanks.
btw, is this a regression? does external windows credential provider never work?

Reply:
Sure.
Since external credential provider is relatively new feature I think (?) and it is also Windows (fewer user), I'm not sure if other cloud is using it.

Reply:
I added a comment there. The reason that I don't add a UT is that UT tests are still run on linux, not the OS of this change.